### PR TITLE
[risk=no] fixing break on demo cohorts

### PR DESCRIPTION
**Pull request by freemabd1**
fixing break on demo cohorts

### CODE REVIEW

File: api/demos/ICD10-F31.30-Bi-polar-female.json [hunk 1]
```diff
@@ -1,63 +1,71 @@
...
             }
-          ],
-          "modifiers": []
+          ]
         }
       ]
     }
-  ]
+  }
 }
```
Comment (blrubenstein) [inline api/demos/ICD10-F31.30-Bi-polar-female.json]: Nit: Newline at end of file.

File: api/demos/28-30-Females.json [hunk 2]
```diff
@@ -1,54 +1,62 @@
...
+          ]
         }
-      ]
+      ],
+      "excludes": []
     }
-  ],
-  "excludes": []
+  }
 }
```
Comment (blrubenstein) [inline api/demos/28-30-Females.json]: Nit: Newline at end of file

File: api/demos/28-29age-toreACL.json [hunk 3]
```diff
@@ -1,42 +1,50 @@
...
           ]
         }
-      ]
+      ],
+      "excludes": []
     }
-  ],
-  "excludes": []
+  }
 }
```
Comment (blrubenstein) [inline api/demos/28-29age-toreACL.json]: Nit: Newline at end of file

Comment (freemabd1) [reply api/demos/28-29age-toreACL.json]: done.

File: api/demos/28-30-Females.json (hunk 2 above)
Comment (freemabd1) [reply api/demos/28-30-Females.json]: done.

File: api/demos/ICD10-F31.30-Bi-polar-female.json (hunk 1 above)
Comment (freemabd1) [reply api/demos/ICD10-F31.30-Bi-polar-female.json]: done.

**Pull request merged**